src/openforms/emails/tasks.py

```diff
@@ -5,13 +5,11 @@
 from django.utils import timezone
 from django.utils.translation import gettext_lazy as _
 
-from django_yubin.models import Message
-
 from openforms.celery import app
 from openforms.config.models import GlobalConfiguration
-from openforms.logging.models import TimelineLogProxy
+from openforms.registrations.utils import collect_registrations_failures
 
-from .utils import send_mail_html
+from .utils import collect_failed_emails, send_mail_html
```

Comment:
can we keep the naming consistent please?

`collection_registration_failures` vs. `collect_failed_emails` -> `collect_failed_registrations` & `collect_failed_emails`